cmd/controller-gen/main.go

```diff
@@ -184,6 +187,7 @@ func main() {
 	cmd.Flags().CountVarP(&whichLevel, "which-markers", "w", "print out all markers available with the requested generators\n(up to -www for the most detailed output, or -wwww for json output)")
 	cmd.Flags().CountVarP(&helpLevel, "detailed-help", "h", "print out more detailed help\n(up to -hhh for the most detailed output, or -hhhh for json output)")
 	cmd.Flags().BoolVar(&showVersion, "version", false, "show version")
+	cmd.Flags().StringSliceVar(&buildTags, "build-tags", nil, "build tags to use when loading Go packages")
```

Comment:
Maybe change to `go-build-tags`?

Reply:
@sbueringer @JoelSpeed if we change the name of the flag than we can move forward here?

Reply:
Maybe `--load-build-tags` and (if necessary in the future) `--generate-build-tags`?